rules/windows/discovery_files_dir_systeminfo_via_cmd.toml

```diff
@@ -50,19 +51,25 @@ If enabling an EQL rule on a non-elastic-agent index (such as beats) for version
 risk_score = 21
 rule_id = "d68e95ad-1c82-4074-a12a-125fe10ac8ba"
 severity = "low"
-tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Discovery", "Tactic: Execution", "Resources: Investigation Guide", "Data Source: Elastic Defend"]
+tags = ["Domain: Endpoint",
+        "OS: Windows",
+        "Use Case: Threat Detection",
+        "Tactic: Discovery",
+        "Tactic: Execution",
+        "Resources: Investigation Guide",
+        "Data Source: Elastic Defend"
+        ]
 timestamp_override = "event.ingested"
-type = "eql"
+type = "new_terms"
 
 query = '''
-process where host.os.type == "windows" and event.type == "start" and
-  process.name : "cmd.exe" and process.args : "/c" and process.args : ("set", "dir") and
-  not process.parent.executable : ("?:\\Program Files\\*", "?:\\Program Files (x86)\\*", "?:\\PROGRA~1\\*")
+host.os.type:windows and event.category:process and event.type:start and process.name.caseless:"cmd.exe" and 
+process.args:("/c" and ("dir" or "set"))
```

Comment:
`process.args.caseless`?

Reply:
Yea unfortunately we cannot use fields such as process.parent.name.caseless, process.parent.executable.caseless, process.args.caseless etc. Apparently, we can use process.command_line.caseless (I did not know, so thanks!). 

This one would for example not execute on `cmd.exe /C ...` due to capitalization. However, the issue with it is, in KQL building queries with spaces and wildcards are a mess, as we can't use quotes like in your example above. What we can do is something like:
```
host.os.type:windows and event.category:process and event.type:start and process.name.caseless:"cmd.exe" and process.command_line.caseless:(
  (cmd.exe?/c* and (cmd.exe*dir* or cmd.exe*set*)) or (C\:\\\\Windows\\\\system32\\\\cmd.exe?/c* and (C\:\\\\Windows\\\\system32\\\\cmd.exe*dir* or C\:\\\\Windows\\\\system32\\\\cmd.exe*set*))
)
```
Or building it into one query if we expect the `set` and `dir` commands to be always used as first argument within the command:
```
host.os.type:windows and event.category:process and event.type:start and process.name.caseless:"cmd.exe" and process.command_line.caseless:(
  (cmd.exe?/c?dir* or cmd.exe?/c?set*) or (C\:\\\\Windows\\\\system32\\\\cmd.exe?/c?dir* or C\:\\\\Windows\\\\system32\\\\cmd.exe?/c?set*)
)
```
What do you think?